src/Components/Trending/Trending.jsx

```diff
@@ -0,0 +1,65 @@
+import "./Trending.scss";
```

Comment:
Told to use module.scss because it removes CSS selector collisions because it adds a unique name for each component.

[Reference](https://blog.bitsrc.io/how-to-use-sass-and-css-modules-with-create-react-app-83fa8b805e5e)

Change it to module.scss and change all the classnames. You can refer to other components in this project as well.